src/citrine/_rest/pageable.py

```diff
@@ -69,14 +70,14 @@ def _fetch_page(self,
         """
         # To avoid setting defaults -> reduce mutation risk, and to make more extensible
         path = self._get_path() if path is None else path
-        fetch_func = self.session.get_resource if fetch_func is None else fetch_func
+        fetch_func = fetch_func or partial(self.session.get_resource, version=self._api_version)
         json_body = {} if json_body is None else json_body
 
         module_type = getattr(self, '_module_type', None)
         params = self._page_params(page, per_page, module_type)
         params.update(additional_params or {})
 
-        data = fetch_func(path, params=params, version=self._api_version, **json_body)
+        data = fetch_func(path, params=params, **json_body)
```

Comment:
So if a fetch_func is provided (not the one you define with partial args) it will always have the api version defined?

Reply:
Also, on a related note, this highlights that our units tests aren't checking the called API version matches what's expected. I vaguely recall knowing that, even purposely maintaining it to avoid needing to overhaul a bunch of tests. But I should look into fixing that...

Reply:
FYI, I created the aforementioned ticket: https://citrine.atlassian.net/browse/PNE-662